CONTRIBUTING-ACL2020.md

```diff
@@ -1,4 +1,106 @@
+nano ~/.bashrc
+# in the ~/.bashrc file
+# modify the following lines accordingly and add them at the end of the file
+export VIRTUALENVWRAPPER_PYTHON='the output of which python3'
+source 'the output of which virtualenvwrapper.sh'
+# close ~./bashrc by pressing ctrl+x and save the changes by pressing "y"
+# refresh the bash profile
+source ~/.bashrc
+# create virtual environment for acl
+mkvirtualenv --python='the output of which python3' your_env_name
+# by now you will have a python3.x virtual environment specificaly for acl-2020
+# if you accidiently quit the current session,
+# you can reactivate the environemnt by the following code
+workon your_env_name
+# install required packages
+pip install -r requirements.txt
+```
+
+
+
+Now the environment is activated.
+
+
+
+## How to Create Pull Request
+
+As mentioned in the [GitHub help documentation](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request), creating a pull request can propose and collaborate on changes to a repository. Those changes will be proposed to a *branch*, which ensures that the ```master``` branch only contains finished and approved works. 
+
+
+
+```shell
+# pull the newest update from the master branch
+git pull
+# create local branch
+git checkout -b your_branch_name
+```
+
+By now you have created a new local branch. Then you can make contributions to the content. Once you've finished, use the following command in the terminal to commit to the local branch.
+
+```shell
+git add . # add all files
+git add file_changed # only add file the named file_changed
+git commit -am 'your update logs'
+git push --set-upstream origin your_branch_name # setup the upstream
+```
+
+Now you can open a pull request on the repository's website. Additionally, on the repository's page, you should click ```New pull request``` button near the ```Branch: master```. Then it will redirect to a new page. On this page, you need to be very carefully for the choices of base repository and head repository. Following the example shown in this document, the head repository should be ```acl-org/acl-2020-virtual-conference```, and ```compare: your_branch_name``` next to the head repository menu. The base repository should be ```acl-org/acl-2020-virtual-conference```.  Then you can click the green button ```Create pull request```. By clicking this button, you have created a new pull request and a collaborator will be notified and review your changes. The collaborator will discuss any further changes for your contributions. Once approved, your branch will be merged into ```master``` branch. Before you make any further contributions, don't forget to ```git pull``` to update your local files.
+
+For more details about opening a pull request on the website, please check the  [GitHub help documentation](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).
+
+
+
+## Additional Instructions
```

Comment:
Can you move this section before "How to create Pull Request"?

Reply:
You mean the ```Additional Instructions```?

Reply:
Oh I mean the whole section below. Probably should put them right after the `git clone`

Reply:
So the structure will be:

- how to contribute to acl-2020
    -  set up environment
        -  get a github repository
        -  install required packages 
    - additional instructions
    - how to create pull request
    - deploy aws with login authentication

Is this correct?